sql-files/main.sql

```diff
@@ -685,6 +685,7 @@ CREATE TABLE IF NOT EXISTS `sc_data` (
   `char_id` INT(11) UNSIGNED NOT NULL,
   `type` SMALLINT(11) UNSIGNED NOT NULL,
   `tick` INT(11) NOT NULL,
+  `total_tick` INT(11) NOT NULL DEFAULT '0',
```

Comment:
Sql upgrade file is not having default value.
So either have it on none or both